### ISSUE TITLE
Sequence Viewer truncates sequences, but scrolls overview.

### ISSUE THREAD

**Issue by Sergey-Knyazev** [CDCgov/MicrobeTrace-Archive#26]
MSA trims the sequences.
Full-screen support would be helpful for the screen.

**Comment by AABoyles** [comment]
Agree, would be nice if we could force it to use a single interface (truncate+local scrolling OR global scrolling).
